lava_test_plans/testcases/master/template-android-xts.jinja2

```diff
@@ -0,0 +1,38 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{% set xts_test_params = xts_test_params|default("xts-not-work") %}
+{## the test_name is a part of the job name ##}
+{% set test_name = test_name|default("xts") %}
+{## the test_suite_name is a part of the path of the lava result ##}
+{% set xts_test_suite_name = xts_test_suite_name|default(test_name) %}
+{% set xts_test_path = xts_test_path|default("xts-path") %}
+{% set xts_pkg_name = xts_pkg_name|default("android-xts.zip") %}
+{% set xts_expect_reboot = xts_expect_reboot|default(false) %}
+{% set xts_result_format = xts_result_format|default("aggregated") %}
+{% set xts_test_definition_path = xts_test_definition_path|default("automated/android/noninteractive-tradefed/tradefed.yaml") %}
+{% set xts_metadata_prefix = xts_metadata_prefix|default('xts') %}
+{% set xts_version = xts_version|default("xts-version-unknown") %}
```

Comment:
A lot of the "dummy defaults" isn't needed here.
Since you said that people should use the template-android-(c|v)ts.jinja2 and not the xtx.jinja2 template.